java/apps/recovery-portal/src/main/resources/org/wso2/carbon/identity/mgt/recovery/endpoint/i18n/Resources.properties

```diff
@@ -129,13 +128,112 @@ error=Error
+Password.reset.Success=Password Reset Successfully
+need.help.contact.us=Need help? Contact us via
+with.tracking.reference.below=with the tracking reference given below.
+Copied=Copied
+tracking.reference.id=Tracking Reference ID
+account.verified.successfully=Account verified successfully
+you.are.almost.there=You're almost there!
+account.created.successfully=Account created successfully
+your.account.with.username=Your account with the username
+has.been.verified.successfully=has been verified successfully.
+you.will.redirected.back.to.the.application.in=You will be redirected back to the application in
+seconds=seconds.
+you.can.now.sign.in.to.account=You can now sign in to your account.
+check.your.inbox.at=Check your inbox at
+for.instructions.to.activate.your.account=for instructions to activate your account.
+several.functionalities.will.not.available.until.confirmation=to confirm your account. Several functionalities will not be available until you confirm the account.
+enter.your.password=Enter your password
+password.requirements.detail=Your password should contain at least eight characters, including at least one each from uppercase and lowercase letters and numbers.
+enter.country=Enter Country
+enter.birth.date=Enter Birth Date
+enter=Enter
+send.reset.link=Send Reset Link
+Proceed=Proceed
+go.back=Go back
+lang.switcher.english=English - United States
+lang.switcher.french=Français - France
+lang.switcher.spanish=Español - España
+lang.switcher.portuguese=Português - Portugal
+lang.switcher.german=Deutsch - Deutschland
+User.identifier=User Identifier
+Callback.url.format.invalid=Configured callback URL format does not match with the provided callback URL in the request
+Invalid.firstname=First name cannot be empty. Please enter a valid first name.
+Invalid.lastname=Last name cannot be empty. Please enter a valid last name.
+Initiate.sign.up=Initiate Sign Up
+Enter.your.user.identifier.here=Enter your user identifier here
 request.has.an.invalid.callback.URL=Request has an invalid callback URL
 invalid.callback.url.found.in.the.request=Invalid callback URL found in the request
-please.contact.administrator.to.fix.issue=Please contact the administrator to fix this issue.
-lang.switcher.english=English - United States
-lang.switcher.french=Fran�ais - France
 select.language=Select Language
+username.length.should.be=The username length should be between
+username.with.symbols=The username should consist of alphanumeric characters (a-z, A-Z, 0-9) and must include at least one letter.
+
+# Password Management
+forgot.password=Forgot Password?
+password.reset.with.username=Don't worry, it happens. We will send you an email to reset your password.
+password.reset.with.email=Don't worry, it happens. Please enter your email address. We will send you an email to reset your password.
+password.requirements=Your password should be more than 8 characters, have atleast one uppercase and lowercase letter and atleast one number.
```

Comment:
There was an effort to refactor these error messages to keep those professional. Could you please check and update accordingly?

cc: @Achintha444

Reply:
As discussed with @Achintha444 , Now customers have the capability to modify this content from the branding text customization section.